src/myaml/converters/utils/line.py

```diff
@@ -37,12 +38,12 @@ def get_indent(string):
     return numOfSpaces//NUM_SPACES_IN_INDENT
 
 
-def get_string(string):
+def get_string(string: str) -> str:
```

Comment:
I need to fix this so that  `get_string(string='- this is a string')` returns 'this is a string'.
These two fixes should fix the above